com.avaloq.tools.ddk.xtext/src/com/avaloq/tools/ddk/xtext/resource/persistence/ResourceLoadMode.java

```diff
@@ -50,6 +52,10 @@
      * elements in the resource contents.
      */
     ASSOCIATIONS,
+    /**

```

Comment:
i don't quite understand why this Constituent is needed. It seems to be tightly coupled to node model one...

Reply:
I split NODE_MODEL into SOURCE and NODE_MODEL since they do correspond to separate entries in the ZIP archive and it makes the processing logic during the load more consistent this way. But we could of course have a special case for this.